public/locales/fr/translation.json

```
@@ -2,60 +2,63 @@
             "hi": "Bonjour",
-            "intro": "Nous sommes heureux que vous soyez connecté. Voici la liste des fonctionnalités auxquelles les utilisateurs connectés ont accès ",
+            "intro": "Nous sommes heureux que vous soyez connecté(e). Les utilisateurs connectés ont accès à davantage de fonctionnalités.",
             "upload_files": "Téléversement de fichiers",
-            "expiration": "Délai d'expiration de secret de 14 ou 28 jour",
+            "expiration": "Délai d'expiration de secret de 14 ou 28 jours",
             "secrets": "Lister et supprimer vos secrets",
-            "more": "Plus de fonctionnalités arrivent ! Merci d'avoir rejoint Hemmelig.app !",
+            "more": "Merci d'avoir rejoint Hemmelig.app !",
             "features": "Fonctionnalités"
         },
         "account": {
             "passwords_does_not_match": "Les mots de passe ne correspondent pas",
             "can_not_update_profile": "Impossible de mettre à jour votre compte utilisateur",
             "can_not_delete": "Impossible de supprimer l'utilisateur",
             "delete_account": "Supprimer votre compte",
-            "do_you_want_delete": "Êtes-vous vraiment sûr devouloir supprimer votre compte ?",
-            "dont_delete_account": "Non, ne le supprime pas",
+            "do_you_want_delete": "Êtes-vous sûr(e) de vouloir supprimer votre compte ?",
+            "dont_delete_account": "Non, ne pas le supprimer",
             "email": "Email",
             "your_password": "Mot de passe actuel",
             "current_password": "Votre mot de passe actuel",
             "update_your_password": "Mettez à jour votre mot de passe",
             "new_password": "Nouveau mot de passe",
-            "confirm_password": "Confirmez le nouveau mot de passe",
+            "confirm_password": "Confirmer le nouveau mot de passe",
             "confirm_new_password": "Confirmez votre mot de passe",
             "update_details": "Mettre à jour"
         },
         "secrets": {
             "id": "Identifiant",
             "expires": "Expire",
             "public": "Public",
-            "delete": "Suppression",
+            "delete": "Supprimer",
             "yes": "Oui",
             "no": "Non",
             "delete_secret": "Supprimer ce secret",
-            "dont_delete_secret": "Non, ne le supprime pas",
-            "do_you_want_delete": "Êtes-vous vraiment sûr de vouloir supprimer ce secret ?"
+            "dont_delete_secret": "Non, ne pas le supprimer",
+            "do_you_want_delete": "Êtes-vous sûr(e) de vouloir supprimer ce secret ?"
         },
         "settings": {
             "read_only_mode": "Mode lecture seule",
-            "readonly_only_for_non_admin": "L'instance Hemmelig doit-elle être en lecture seule pour les utilisateurs non administrateurs ou créateur ?",
+            "readonly_only_for_non_admin": "L'instance Hemmelig doit-elle être en lecture seule pour les utilisateurs non administrateurs ?",
```

The English version reads "Should the Hemmelig instance be read only for non admin users?", whereas the French version mentioned "non admin **or creator** users". I removed the creator thing to align it with English.

---

According to the README, creators are allowed to create secrets on a read-only instance, so I guess I should revert my change in French and fix the English?

---

Correct. Creators are allowed to create secrets.

---

OK, thanks. Let me revert that then (and fix a grammar issue at the same time):

```suggestion
            "readonly_only_for_non_admin": "L'instance Hemmelig doit-elle être en lecture seule pour les utilisateurs non administrateurs ou créateurs ?",
```